python/tests/lang/definitions/test_schema.py

```diff
@@ -50,14 +50,14 @@
 class TestDefinitionSchemas(TestCase):
     def test_get_root_schema_definitions_with_only_core_spec(self):
         test_context = get_core_spec_context()
-        core_root_fields = get_root_fields()
+        core_root_definitions = get_root_definitions()
 
         actual_results = get_root_schema_definitions(test_context)
 
-        self.assertGreater(len(core_root_fields), 0)
-        for root_field in core_root_fields:
-            root_name = root_field.get(DEFINITION_FIELD_NAME)
-            root_type = root_field.get(DEFINITION_FIELD_TYPE)
+        self.assertGreater(len(core_root_definitions), 0)
+        for root_definition in core_root_definitions:
+            root_name = root_definition.get_root()
+            root_type = root_definition.name
```

Comment:
We can certainly change the variable names, I was just trying to minimize changes.  If you think it is important, insert the recommended code change and I'll add it to the commit batch for your review.

Reply:
I'm not confident enough in which route is best, because as I said in the first part of my comment it helps to confirm/explain how `root_definition.name` is now functioning as `root.get("type") `. And I know if I'm not confident I am following logic within the codebase, one of my stops is tests to see how the piece I am confused about operates. So, I don't want to lose that prompting.

But the variable names tripped me up. Looking at it this morning versus last night does make me think part of that was due to conducting a review late in the day.

I think we leave it to help provide operational awareness so that within the main codebase the role of `root.name` is more easily understood.